src/go/k8s/kind.yaml

```diff
@@ -11,3 +11,11 @@ nodes:
   image: vectorizedcloud/kindestnode:v1.21.1-pr2515
 - role: worker
   image: vectorizedcloud/kindestnode:v1.21.1-pr2515
+- role: worker
```

Comment:
😅 oh wow. Can it handle it? :D I think we had some problems with bigger clusters

Reply:
Yeah, this was the biggest I could get. I ran with 16 and 4 parallel tests on my machine and it worked, but it didn't save much more time than this with 2 parallel does.